### PR TITLE
Update ledgers

### PR DESCRIPTION
**Pull request by smyzin**
- updated packages
- updated ledgers

### CODE REVIEW

File: src/helpers/contracts/getLedgerRecord.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export default function (original: string, derivative: string) {
```
Comment (upacyxou) [inline src/helpers/contracts/getLedgerRecord.ts:1]: this function and `getEmailLedgerRecord.ts` from above looks very similar, do we actually need two different functions for this?

Comment (smyzin) [reply src/helpers/contracts/getLedgerRecord.ts]: Updated. Thank you!

File: src/helpers/contracts/getEmailLedger.ts [hunk 2]
```diff
@@ -1,18 +1,15 @@
 import { SCEmailLedger } from '@big-whale-labs/seal-cred-ledger-contract'
 import EmailLedger from 'models/EmailLedger'
-import getEmailLedgerRecord from 'helpers/contracts/getEmailLedgerRecord'
+import getLedgerRecord from 'helpers/contracts/getLedgerRecord'
 
 export default async function (ledger: SCEmailLedger): Promise<EmailLedger> {
-  const eventsFilter = ledger.filters.CreateDerivativeContract()
+  const eventsFilter = ledger.filters.CreateDerivative()
```
Comment (T-Damer) [inline src/helpers/contracts/getEmailLedger.ts:6]: Seems like a repetitive code, I'll extract this into function

File: src/models/EmailLedger.ts [hunk 3]
```diff
@@ -1,6 +1,6 @@
 export default interface SCEmailLedger {
-  [domain: string]: {
+  [originalContract: string]: {
+    originalContract: string
```
Comment (T-Damer) [inline src/models/EmailLedger.ts:3]: Why do we need to map `originalContract` to `originalContract` if we can use the `key`?

**Review by T-Damer — APPROVED** [review]
lgtm

**Pull request merged**